source/javascripts/core/models/BitriseYmlService.ts

```diff
@@ -50,6 +70,22 @@ function createWorkflow(workflowId: string, yml: BitriseYml, baseWorkflowId?: st
   return copy;
 }
 
+function updateWorkflow(workflowId: string, workflow: WorkflowYmlObject, yml: BitriseYml): BitriseYml {
+  const copy = deepCloneSimpleObject(yml);
+
+  mapValues(workflow, (value: string, key: never) => {
```

Comment:
Why is the type of the `key` `never`?

Reply:
Because it's hard to type properly and it is not important here what is the type of this.